sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -631,6 +648,108 @@ protected void addFields() {
 			I18nProperties.getPrefixDescription(PersonDto.I18N_PREFIX, PersonDto.ADDITIONAL_DETAILS, "") + "\n"
 				+ I18nProperties.getDescription(Descriptions.descGdpr));
 		CssStyles.style(additionalDetails, CssStyles.CAPTION_HIDDEN);
+
+		hasGuardian = addField(PersonDto.HAS_GUARDIAN, CheckBox.class);
+		hasGuardian.addValueChangeListener(e -> updateHasGuardianCheckBox());
+
+		isIncapacitated = addField(PersonDto.IS_INCAPACITATED, CheckBox.class);
+		isIncapacitated.addValueChangeListener(e -> updateIsIncapacitatedCheckBox(true));
+
+		isEmancipated = addField(PersonDto.IS_EMANCIPATED, CheckBox.class);
+		isEmancipated.addValueChangeListener(e -> onEmancipatedChange());
+		hasGuardian.setEnabled(false);
+		minimumAdultAge = FacadeProvider.getConfigFacade().getMinimumAdultAge();//2 places
+		minimumEmancipatedAge = FacadeProvider.getConfigFacade().getMinimumEmancipatedAge();
+	}
+
+	private boolean canBeEmancipated() {
+		int approximateAge = getApproximateAge(ApproximateAgeType.YEARS);
+		if (approximateAge == -1){
+				return false;
+		}
+		return approximateAge >= minimumEmancipatedAge && approximateAge < minimumAdultAge;
+	}
+
+	private int getApproximateAge(ApproximateAgeType approximateAgeType) {
+		Date birthDate = calcBirthDateValue();
+		if (birthDate != null) {
+			Pair<Integer, ApproximateAgeType> pair =
+					ApproximateAgeHelper.getApproximateAge(birthDate, (Date) getFieldGroup().getField(PersonDto.DEATH_DATE).getValue());
+			if ((pair.getElement0() != null) && (pair.getElement1() == approximateAgeType)) {
+				return pair.getElement0();
+			}
+		}
+		return -1;
+	}
+
+	private void onEmancipatedChange() {
+		boolean isEmancipatedHidden = (isEmancipated == null) || (!isEmancipated.getValue());
+		boolean isEmancipatedChecked = (isEmancipated != null) && (isEmancipated.getValue());
+		hasGuardian.setValue(isEmancipatedHidden);
+		if(isEmancipatedHidden || isEmancipatedChecked) {
+			nameOfGuardians.setValue("");
+		}
+		updateHasGuardianCheckBox();
+	}
+
+	private void updateIsIncapacitatedCheckBox(boolean isInitialized) {
+		boolean isIncapacitatedChecked = (isIncapacitated == null) || (isIncapacitated.getValue());
+		isEmancipated.setVisible(!isIncapacitatedChecked && canBeEmancipated());
```

Comment:
"can be emancipated" is used in a single place. You can integrate it in this method. Create a variable "aproximateAge" which can be used both for calculating "canBeEmancipated" and for the if clause below.